package.json

```diff
@@ -242,6 +243,6 @@
   },
   "engines": {
     "node": ">=14.15.0 <15.0.0",
-    "npm": ">=6.14.0 <8.0.0"
+    "npm": ">=7.14.0 <8.0.0"
```

Comment:
by allowing the use of `npm` older than `7.0.0` we're getting thrashing of old and new `package-lock.json` files